people/zuzana.json

```diff
@@ -8,7 +8,7 @@
 
   "email": "zuzana.cervinova@mild.blue",
 
-  "title_cs": "JUNIOR FRONTEND DEVELOPER",
+  "title_cs": "JUNIOR FRONTEND VÝVOJÁŘKA",
 
   "title_en": "JUNIOR FRONTEND DEVELOPER",
```

Comment:
Dala bych normální písmo, ne uppercase. Třeba to někdy na web změníme a jen Zuzka to bude mít natvrdo kapitalkama

Reply:
Stejně u ostatních lidi

Reply:
Určite dobrý postreh, upraví @chirimiri22 , dík,

Reply:
Miri opravila.